### PR TITLE
stressng integration

### PR DESCRIPTION
**Pull request by mkarg75**
ssia

### CODE REVIEW

File: roles/stressng/templates/stressng_workload.yml.j2 [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+            value: "{{ prometheus.prom_url | default() }}"
+{% endif %}
+        command: ["/bin/sh", "-c"]
+        args: 
+          - "export runtype={{workload_args.runtype}};
+             export timeout={{workload_args.timeout}};
+             export vm_stressors={{workload_args.vm_stressors}};
+             export vm_bytes={{workload_args.vm_bytes}};
+             export mem_stressors={{workload_args.mem_stressors}};
+             run_snafu --tool stressng -u {{ uuid }}"
```
Comment (rsevilla87) [inline roles/stressng/templates/stressng_workload.yml.j2:59]: Append -j /workload/jobfile

**Review by rsevilla87 — CHANGES_REQUESTED** [review]

File: docs/stressng.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      cpu_percentage: "100"
+      # vm stressor option
+      vm_stressors: "1"
+      vm_bytes: "128M"
+      # mem stressor options
+      mem_stressors: "1"
+
+```
+
+The stressng benchmark is divided into 3 subsystems, driven by so-called stressors. In the above example we have cpu stressors, virtual machine (vm) stressors and memory stressors. They are running in a parallel fashion, but could also run sequentially.
```
Comment (smalleni) [inline docs/stressng.md:44]: Did you mean virtual memory instead?

**Comment by rsevilla87** [comment]
@mkarg75 there're some conflicts in README.md, can you fix them?

**Comment by mkarg75** [comment]
Should be sorted.

File: tests/test_crs/valid_stressng.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+apiVersion: ripsaw.cloudbulldozer.io/v1alpha1
+kind: Benchmark
+metadata:
+  name: stressng
+  namespace: my-ripsaw
+spec:
+  elasticsearch:
+    server: search-cloud-perf-lqrf3jjtaqo7727m7ynd2xyt4y.us-west-2.es.amazonaws.com
+    port: 80
```
Comment (rsevilla87) [inline tests/test_crs/valid_stressng.yaml:9]: Hey @mkarg75, can you replace these two by:

```suggestion
    server: ES_SERVER
    port: ES_PORT
```
These names are replaced by the CI.

**Review by rsevilla87 — CHANGES_REQUESTED** [review]

File: roles/stressng/tasks/main.yaml [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+          jobfile: "{{ lookup ('template', 'jobfile.j2') }}"
+
+  - operator_sdk.util.k8s_status:
+      api_version: ripsaw.cloudbulldozer.io/v1alpha1
+      kind: Benchmark
+      name: "{{ meta.name }}"
+      namespace: "{{ operator_namespace }}"
+      status:
+        state: ConfigMaps Created
+        complete: true
```
Comment (dry923) [inline roles/stressng/tasks/main.yaml:47]: this line should say complete: false or be removed

File: README.md [hunk 5]
```diff
@@ -20,6 +20,7 @@ a performance baseline of Kubernetes cluster on your provider.
 | [Service Mesh](docs/servicemesh.md) | Microservices     | Working            |  Used, default : 3second   | Not Supported         | Not Supported   |
 | [Vegeta](docs/vegeta.md)       | HTTP Performance       | Working            |  Used, default : 3second  | Not Supported          | Not Supported   |
 | [Scale Openshift](docs/scale_openshift.md) | Scale Openshift Cluster       | Working            |  Used, default : 3second  | Not Supported         | Not Supported  |
+| [stressng](docs/stressng.md)   | Stress system resources | Working            |                 | Not Supported         | Not Supported  |
```
Comment (dry923) [inline README.md:23]: Can you add "Used, default : 3second" for the reconciliation usage section since you have added that?

**Review by rsevilla87 — APPROVED** [review]
LGTM!

**Review by dry923 — APPROVED** [review]
LGTM!

**Pull request merged**